app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> {order created_at: :desc}
```

Comment:
you shouldn't use `default_scope`
Please, you should define scope `newest` or `lastest` for order

Reply:
@VuThiTranVan what do you mean ?